### PR TITLE
0.2.4 Main

### PR DESCRIPTION
**Pull request by JamesRobertsonGames**
[multipart-feedstock](https://github.com/AnacondaRecipes/multipart-feedstock) 0.2.4

**Destination channel:** defaults

### Links

- [PKG-4578](https://anaconda.atlassian.net/browse/PKG-4578) 
- [Upstream repository](https://github.com/defnull/multipart)
- [Upstream changelog/diff](https://github.com/defnull/multipart/releases/tag/v0.2.4)
- Relevant dependency PRs:
  - https://github.com/AnacondaRecipes/moto-feedstock/pull/36

### Explanation of changes:
- initial recipe

[PKG-4578]: https://anaconda.atlassian.net/browse/PKG-4578?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+build:
+  skip: true # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:22]:
```suggestion
    - setuptools
    - wheel
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM but `setuptools`

**Review by cbouss — COMMENTED** [review]
Can you make the feedstock public and add a feedstock license and readme?
You can copy/paste from https://github.com/AnacondaRecipes/datadog-api-client-feedstock

**Comment by skupr-anaconda** [comment]
`.gitattributes` and `.gitignore` are also nice to have. See examples:
- https://github.com/AnacondaRecipes/sspilib-feedstock/blob/89f7960fc42d9c5736a171889ad083cfdf951c2e/.gitattributes
- https://github.com/AnacondaRecipes/sspilib-feedstock/blob/89f7960fc42d9c5736a171889ad083cfdf951c2e/.gitignore

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+
+test:
+  source_files:
+    - test
+    - multipart.py
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: This is suspicious @JamesRobertsonGames. It should not have to be copied.

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (JamesRobertsonGames) [reply recipe/meta.yaml]: Is it? Usually I would do the source folder however this is just an individual file not contained in a folder

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml]: `multipart.py` is the `multipart` module. It should already be part of the conda package automatically installed during the tests.

Comment (JamesRobertsonGames) [reply recipe/meta.yaml]: Oh I see, it works without the source files in this instance

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Comment by cbouss** [comment]
It would still be good to add a license file for the feedstock before merging.

File: LICENSE
Comment (JeanChristopheMorinPerso) [inline LICENSE:1]: @JamesRobertsonGames This license should be moved into the `recipe` folder. The feedstock should have a different license.

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: LICENSE
Comment (JeanChristopheMorinPerso) [reply LICENSE:1]: And technically, the upstream package already has its own license, https://github.com/defnull/multipart/blob/master/LICENSE.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**